package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@stencil/core",
   "version": "3.2.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
Ah, good old NPM v9 bumped the lockfile version for us - that explains the number of lines changed 🙃